openapi.yaml

```diff
@@ -10615,6 +10618,300 @@ components:
+                  sub_type: quick_reply
+                  payload: Yes-Button-Payload
+                - index: '1'
+                  sub_type: quick_reply
+                  payload: No-Button-Payload
+        WhatsApp Message Using Location and Link Button (static url):
+          to: '639177176978'
+          subject: message subject
+          whatsapp:
+            type: icebreaker
+            name: location_cta_staticurl
+            content:
+              header:
+                location:
+                  longitude: -122.425332
+                  latitude: 37.758056
+                  name: Facebook HQ
+                  address: '1 Hacker Way, Menlo Park, CA 94025'
+              body:
+                - text: John
+                - text: '619704512462'
+        'WhatsApp Message Using Link button (dynamic) ':
+          to: '61430933333'
+          subject: message subject
+          whatsapp:
+            type: icebreaker
+            name: new_media_message_rules
+            content:
+              header:
+                image:
+                  link: 'http://placekitten.com/200/300'
+              body:
+                - text: John
+              buttons:
+                - index: '0'
+                  sub_type: url
+                  text: search?q=Whisprr
+        WhatsApp Message Using Link button (static url + call phonenumber):
+          to: '61430933333'
+          subject: message subject
+          whatsapp:
+            type: icebreaker
+            name: hsbc_alert
+            content:
+              body:
+                - text: John
+                - text: Everything is connected!
+                - text: 'https://hsbc.com/renewal'
+                - text: 'https://hsbc.com/terms'
+                - text: 'https://hsbc.com/privacy'
+      properties:
+        type:
+          type: string
+          description: |-
+            The type of WhatsApp message to be sent, e.g. "template".
+
+            Whispir currently only supports type "template".
+          example: 'icebreaker'  
+        name:
+          type: string
+          description: |-
+            The name of the template to be sent 
+            > Must be the same as how it appears in your WhatsApp Business Manager
+          example: 'account_update_example_image_p'  
+        content:
+          $ref: '#/components/schemas/WAContent'
+      required:
+        - type
+        - name
+        - content
+    WAContent:
+      title: WAContent
+      x-stoplight:
+        id: fklmr2skqadi5
+      type: object
+      description: 'The content of your message which may contain any text and media that will be sent to the recipient. The contents can include a header, a body, and buttons.'
+      properties:
+        header:
+          $ref: '#/components/schemas/WAContentHeader'
+        body:
+          type: array
+          description: The contents of your WhatsApp message
+          items:
+            title: Object
+            type: object
```

Comment:
What are the key/value pairs required for this object? The `object` type doesn't make it clear to developers what is required, we need to provide the `properties` to specify the requirements here.

Reply:
Added properties to the body object